src/style/base.less

```diff
@@ -1,4 +1,4 @@
-@import url('/fonts/lato.css');
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700,900');
```

Comment:
can you package this with the app so they dont have to get it remotely? could be an issue with China users trying to access google apis.

Reply:
souga, updated